python/python/benchmarks/test_scan.py

```diff
@@ -140,5 +140,5 @@ def test_filter_multiple(benchmark, sample_dataset):
         filter="i > 1000 and i < 5000 and s in ('hello', 'world')",
     )
 
-    assert result.num_rows == 1
+    assert result.num_rows > 1
```

Comment:
Isn't this odd that this would change? I don't see a corresponding change in the query or data.

Reply:
This assert fails for me on `main`.  I'm not sure if it ever passed (note that benchmark results are still reported even though it fails).  Looking at the logic I wouldn't expect to get a single result.  The # of results appears to be random (based on the random.choice in the string column)

Reply:
Oh okay 🤦  Thanks for explaining.